apps/sms/views/conversation/js/conversation.js

```diff
@@ -2172,7 +2172,7 @@ var ConversationView = {
 
   sendMessage: function conv_sendMessage(opts) {
     var content = Compose.getContent(),
-        subject = Compose.getSubject(),
+        subject = Compose.isSubjectVisible ? Compose.getSubject() : null,
```

Comment:
I really would prefer that Compose.js has this in Compose.getSubject() (it already has such a check in Compose.hasSubject() btw...)